extension/js/common/composer.ts

```diff
@@ -628,7 +628,7 @@ export class Composer {
   private extractProcessSendMsg = async () => {
     try {
       const recipients = this.getRecipientsFromDom();
-      const subject = this.v.subject || String($('#input_subject').val()); // replies have subject in url params
+      const subject = this.v.subject || ($('#input_subject').val() === undefined ? '' : String($('#input_subject').val())); // replies have subject in url params
```

Comment:
We could test for truthiness here eg `$('#input_subject').val() ? String($('#input_subject').val()) : '')` or shorter `String($('#input_subject').val() || '')`. 

This is a nitpick of course, I'll approve it as is.

Reply:
We can't use `String($('#input_subject').val())` as a boolean because `String(undefined)` evaluates to `"undefined"`, which is not falsy!

Reply:
What you say is correct, but that's not what I proposed :)
`String($('#input_subject').val())`
vs
`String($('#input_subject').val() || '')`

Reply:
Ah misread it. That would be a little more elegant!